tools/topology/m4/pcm.m4

```diff
@@ -87,7 +101,9 @@ define(`PCM_PLAYBACK_ADD',
 `'
 `		capabilities STR($3)'
 `	}'
-`}', `fatal_error(`Invalid parameters ($#) to PCM_PLAYBACK_ADD')')'
+`'
+`	compress STR($4)'
```

Comment:
This change (adding the "compress" field) might be an ABI change? Does it build?

Reply:
(compress "false" wasn't there before on non-compress situation so I am a little concerned about if the driver itself supports this)

Reply:
Yes, all m4 topologies files build fine. There is no ABI change AFAICT.

alsa-lib/src/topology/pcm.c code that creates the binary .tplg file from m4 (.conf) files does the following:

```
                if (strcmp(id, "compress") == 0) {
                        if (snd_config_get_string(n, &val) < 0)
                                return -EINVAL;

                        if (strcmp(val, "true") == 0)
                            pcm->compress = 1;

                        tplg_dbg("\t%s: %s\n", id, val);
                        continue;
                }

Reply:
So the driver already has support for parsing this. Fair enough.